### ISSUE TITLE
blank_genome: an alternative code

### ISSUE THREAD

**Issue by qingjian1991** [caravagnalab/CNAqc#13]
Hi:

I made some changes on function **blank_genome()** for better visualizations on CNV plot, please review my code.

The figure before changes, The 1q has a complex CNAs changes:

![image](https://user-images.githubusercontent.com/19546092/139669597-346e3d68-8e91-422b-86cc-7b2da34ecbfb.png)

The figure after changes:

![image](https://user-images.githubusercontent.com/19546092/139669218-2929ae1b-1e03-47a9-b299-9983eab779a6.png)

The changed code is here:
[blank_genome.R](https://github.com/caravagnalab/CNAqc/files/7453130/blank_genome.txt)

Thanks.

Qingjian Chen

**Comment by caravagn** [comment]
Hi @qingjian1991 ,

Thanks for the message.

Is the only change that you are doing this?

```
    #set the chr names in the centromer positions.
    scale_x_continuous(
      breaks = c(0, reference_coordinates$centromerStart, upp),
      labels = c("", gsub(pattern = 'chr', replacement = '', reference_coordinates$chr), "")
    )
```

Could you show with some plots how does this visualise when you do not follow the circular layout (i.e., as a single sample)?

Btw, we'd have to go for a PR if we accept this modification.

**Comment by qingjian1991** [comment]
> Is the only change that you are doing this?

I made two main changes to this code.  One is as your notice.  

Another change is here. The origin code used dashed lines to separate the chromosomes and solid lines to indicate the positions of the centromere. The modified code changed the orders, by using the solid lines to separate the chromosomes and dashed lines to indicate the position of the centromere.

![image](https://user-images.githubusercontent.com/19546092/140450588-d26407e6-7340-4d8e-9fbf-b8aee0141520.png)

**Comment by caravagn** [comment]
I think it is OK, open a PR if you want me to get this into the code.

**Comment by caravagn** [comment]
Merged #16 thanks @caravagn
